app/assets/javascripts/cable/subscriptions/notifications.js

```diff
@@ -0,0 +1,12 @@
+App.cable.subscriptions.create('NotificationsChannel', {
+  received: function(data) {
+    if(data.new_matches_count == 1) {
+      $('.modal .new-matches-count').html('<h3>You have a new match!</h3>');
```

Comment:
Instead of preparing the content on this side, it'd be better to create a partial that renders it and send it as a string through the channel. You can handle the "match/matches" based on the count using the `pluralize` method.